tests/cupy_tests/creation_tests/test_basic.py

```diff
@@ -35,6 +41,12 @@ def test_empty_int(self, xp, dtype, order):
         a.fill(0)
         return a
 
+    @testing.slow
+    def test_empty_huge_int(self):
+        a = cupy.empty(2 ** 31, dtype='b')
+        a.fill(0)
+        self.assertFalse(a.any())
+
```

Comment:
What is the meaning of `_int` suffix? Actual `dtype` is not integer (also true for some other test cases in this file).

Additionally, I suggest covering these cases:
1. It's better to additionally test on int32 and float32, not only boolean.
2. It's better to additionally test on `cupy.ones()` and/or `a.fill(n)` where `n != 0`, because `a.fill(0)` is treated differently from other values.